build.sbt

```diff
@@ -505,7 +506,6 @@ lazy val blazegraph = project
 
 lazy val storagePlugin = project
   .in(file("delta/plugins/storage"))
-  .enablePlugins(JavaAgent)
   .settings(shared, compilation, assertJavaVersion, kamonSettings, coverage, release)
```

Comment:
why is there a need for `kamonSettings`?

Reply:
We need at least `kamon-akka-http` in order to use `operationName` in the routes.